src/ngraph/builder/quantization.cpp

```diff
@@ -121,23 +121,50 @@ namespace ngraph
                                            std::shared_ptr<Node> max_freezed_output,
                                            const bool with_relu)
         {
+            auto output_et = with_relu ? element::u8 : element::i8;
             auto requantization_scale = quantization_util::get_scale(min_input,
                                                                      max_input,
                                                                      min_filter,
                                                                      max_filter,
                                                                      min_freezed_output,
-                                                                     max_freezed_output);
+                                                                     max_freezed_output,
+                                                                     output_et);
 
-            return make_shared<op::QuantizedConvolutionBias>(input,
-                                                             filters,
-                                                             bias,
-                                                             window_movement_strides,
-                                                             window_dilation_strides,
-                                                             padding_below,
-                                                             padding_above,
-                                                             data_dilation_strides,
-                                                             requantization_scale,
-                                                             with_relu);
+            if (bias->get_element_type() != element::i32)
+            {
+                auto zero = make_constant(element::i32, min_input->get_shape(), 0);
+                AxisSet quantization_axes;
+                auto bias_scale =
+                    quantization_util::get_bias_scale(min_input, max_input, min_filter, max_filter);
+                op::Quantize::RoundMode round_mode =
+                    op::Quantize::RoundMode::ROUND_NEAREST_TOWARD_EVEN;
```

Comment:
How do you know the rounding mode?  Shouldn't this match the rounding mode used for the input?

Reply:
It should be the responsibility of the bridge to quantize bias / make bias input conform to the builder/op type checking.  That said, I won't hold up the PR for it provided that... 1) type checking for experimental quantized ops and related testing (test/type_prop.cpp) is included in a separate PR and 2) that this code is moved to the bridge in separate PR.  NOTE:  a separate builder can be used for #2 if the code to do the conversion is too complex for the bridge create on its own.